### PR TITLE
Uncomment resolution steps that have been fixed by changes to the test framework

### PR DESCRIPTION
**Pull request by alexjpwalker**
## What is the goal of this PR?

To expand our resolution tests.

## What are the changes implemented in this PR?

We uncommented the resolution steps that previously had to be commented due to test framework bugs, which were fixed in https://github.com/graknlabs/grakn/pull/5823.

### CODE REVIEW

**Comment by alexjpwalker** [comment]
Note: This PR depends on https://github.com/graknlabs/grakn/pull/5823

File: behaviour/graql/reasoner/resolution/value-predicate.feature [hunk 1]
```diff
@@ -79,8 +78,7 @@ Feature: Value Predicate Resolution
         $a > 0;
       },
       then {
-        $x has is-old $t;
-        $t true;
+        $x has is-old true;
```
Comment (flyingsilverfin) [inline behaviour/graql/reasoner/resolution/value-predicate.feature:81]: was this the format of the test originally?

**Comment by flyingsilverfin** [comment]
Very Cool! Just one question then i'll approve :)

File: behaviour/graql/reasoner/resolution/value-predicate.feature (hunk 1 above)
Comment (alexjpwalker) [reply behaviour/graql/reasoner/resolution/value-predicate.feature:81]: This was one of the tests that we had to modify in order to make it pass, and was one of the incentives for this PR

Comment (flyingsilverfin) [reply behaviour/graql/reasoner/resolution/value-predicate.feature:81]: good

**Pull request merged**